typings/django/db/models/fields/related.pyi

```diff
@@ -275,15 +275,15 @@ class OneToOneField(Generic[_M], RelatedField[_M, _M]):
     @overload
     def __get__(self: _F, instance: Any, owner: Any) -> _F: ...
 
-class ManyToManyField(RelatedField[Sequence[Any], RelatedManager[Any]]):
+class ManyToManyField(Generic[_M], RelatedField[Sequence[_M], RelatedManager[_M]]):
```

Comment:
Looks good!

Could we add a couple examples in:

https://github.com/sbdchd/django-types/blob/main/tests/trout/models.py#L180

to ensure the type checkers don't do anything funny

something like this (which I stole from the django docs)

```python
from django.db import models

class Person(models.Model):
    name = models.CharField(max_length=50)

class Group(models.Model):
    name = models.CharField(max_length=128)
    members = models.ManyToManyField(Person)
```